### PR TITLE
Add standalone helper for reflection services

### PR DESCRIPTION
**Pull request by johanbrandhorst**
Also add some helpful documentation for the new method.

### CODE REVIEW

**Comment by johanbrandhorst** [comment]
Bump @jhump?

File: handlers.go [hunk 1]
```diff
@@ -90,8 +90,8 @@ func RPCInvokeHandler(ch grpcdynamic.Channel, descs []*desc.MethodDescriptor) ht
 	})
 }
 
-// RPCInvokeHandler returns an HTTP handler that can be used to invoke RPCs. The
-// request includes request data, header metadata, and an optional timeout.
+// RPCMetadataHandler returns an HTTP handler that can be used to get metadata
```
Comment (jhump) [inline handlers.go]: Doh! Thanks for fixing!

File: go.sum [hunk 2]
```diff
@@ -3,6 +3,7 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/client9/misspell v0.3.4/go.mod h1:qj6jICC3Q7zFZvVWo7KLAzC3yx5G7kyvSDkc90ppPyw=
 github.com/fullstorydev/grpcurl v1.2.1 h1:5OUMseLFPJ7dXyvav6cB9z5HMr/vRdjeB4+LL9oGl4w=
 github.com/fullstorydev/grpcurl v1.2.1/go.mod h1:R6HR6EI02cafd3StLYD1Stwvqg3YaQCiXD8yj3Laq6g=
+github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b h1:VKtxabqXZkF25pY9ekfRL6a582T4P37/31XEstQ5p58=
```
Comment (jhump) [inline go.sum]: I think the change to this file can be reverted, right?

Comment (johanbrandhorst) [reply go.sum]: I'm not confident enough with modules to say, but I've reverted it.

**Pull request merged**